fsw/src/cs_compute.c

```diff
@@ -316,33 +316,38 @@ int32 CS_ComputeApp(CS_Res_App_Table_Entry_t *ResultsEntry, uint32 *ComputedCSVa
     uint32  NewChecksumValue        = 0;
     int32   Status                  = CS_SUCCESS;
     int32   Result;
-    int32   ResultGetAppID     = CS_ERROR;
-    int32   ResultGetAppInfo   = CS_ERROR;
-    int32   ResultAddressValid = false;
+    int32   ResultGetResourceID   = CS_ERROR;
+    int32   ResultGetResourceInfo = CS_ERROR;
+    int32   ResultAddressValid    = false;
 
     /* variables to get applications address */
-    CFE_ES_AppId_t   AppID = CFE_ES_APPID_UNDEFINED;
+    CFE_ResourceId_t ResourceID;
```

Comment:
Some static analysis checkers might complain about potential uninitialized variable use, err on the side of initializing, especially for something typically set by an API call.